### PR TITLE
GOB: Detection improvements

### PR DESCRIPTION
**Pull request by BJNFNE**

### CODE REVIEW

**Comment by lephilousophe** [comment]
Hello,

I don't understand why these changes couldn't get in #5618.
I asked there if there was more changes planned and you told me it was finished.
And now there is this new PR.

File: engines/gob/gameidtotype.h [hunk 1]
```diff
@@ -54,6 +54,7 @@ static const GameIdToType gameIdToType[] = {
 	{                   "inca2", kGameTypeInca2 },
 	{                "woodruff", kGameTypeWoodruff },
 	{                 "dynasty", kGameTypeDynasty },
+	{                 "dynastywood", kGameTypeDynastyWood },
```
Comment (bluegr) [inline engines/gob/gameidtotype.h:57]:
```suggestion
	{             "dynastywood", kGameTypeDynastyWood },
```

File: engines/gob/gameidtotype.h [hunk 2]
```diff
@@ -73,6 +74,7 @@ static const GameIdToType gameIdToType[] = {
 	{                 "adibou1", kGameTypeAdibou1 },
 	{                 "adibou2", kGameTypeAdibou2 },
 	{                 "adibou3", kGameTypeNone },
+	{            "adiboucuisine", kGameTypeNone },
```
Comment (bluegr) [inline engines/gob/gameidtotype.h:77]:
```suggestion
	{           "adiboucuisine", kGameTypeNone },
```

**Comment by bluegr** [comment]
Thanks!

**Pull request merged**